File: templates/template.py
# run on python 3.7.2
from sys import stdin


def main():
    n = int(stdin.readline())
    s = stdin.readline()
    x = [None] * n
    y = [None] * n
    for i in range(n):
        x[i], y[i] = map(int, stdin.readline().split())
    # solve here
    print(n)
    print(s)
    print(x[0])
    print(y[n - 1])


def solve(n):
    return 0


main()

File: templates/template.php
<?php

// run on php 7.2.13

function println($s)
{
    print $s . PHP_EOL;
}

function main()
{
    $n = trim(fgets(STDIN));
    $s = trim(fgets(STDIN));
    $x = array_fill(0, $n, 0);
    $y = array_fill(0, $n, 0);
    foreach (range(0, $n - 1) as $i) {
        list($x[$i], $y[$i]) = explode(' ', trim(fgets(STDIN)));
    }

    // solve here

    println($n);
    println($s);
    println($x[0]);
    println($y[$n - 1]);
}

function solve(int $n)
{
    return 0;
}

main();